options.go

```diff
@@ -395,6 +402,20 @@ func (o *Options) CLI() serpent.OptionSet {
 			Description: "Coder agent subsystems to report when forwarding logs. " +
 				"The envbuilder subsystem is always included.",
 		},
+		{
+			Flag:  "push-image",
+			Env:   WithEnvPrefix("PUSH_IMAGE"),
+			Value: serpent.BoolOf(&o.PushImage),
+			Description: "Flag to determine if the image should be pushed to " +
+				"the container registry. This option implies reproducible builds.",
+		},
+		{
+			Flag:  "get-cached-image",
+			Env:   WithEnvPrefix("GET_CACHED_IMAGE"),
+			Value: serpent.BoolOf(&o.GetCachedImage),
+			Description: "Flag to determine if the cached image is available, " +
+				"and if it is, to return it.",
```

Comment:
Nit: simplify wording
```suggestion
			Description: "Print the digest of the cached image, if available.",
```